ClementChaumel/ubuntu.com
# Store checkout states in local storage

## Done

- Store checkout states in local storage 
- Remove stored local storage after purchasing 

## QA

- Go to `/pro/subscribe`
- Select items
- Refresh page or log in and see the selected items remains as selected.


## Issue / Card

Fixes https://warthogs.atlassian.net/jira/software/c/projects/WD/boards/801?modal=detail&selectedIssue=WD-304

![ezgif com-gif-maker (18)](https://user-images.githubusercontent.com/57550290/201078683-e891ea89-c38f-4484-8685-1196515b8eee.gif)
